create-quasar/index.d.ts

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
This one is only needed to make `await import('../index.js');` in `create-quasar/scripts/create-test-project.ts` to work without TS errors, right?

Can you add a comment making it explicit?

Reply:
After rebasing, probably due to TS v5, it was no longer needed. So, I removed the .d.ts file altogether.